src/constants/default-state.ts

```diff
@@ -38,9 +39,10 @@ export const DEFAULT_STATE: State = {
   format: false,
   gist: null,
   lastPosition: 0,
-  logs: false,
+  logs: true,
```

Comment:
What does this change do?

Reply:
![bug](https://user-images.githubusercontent.com/35191225/54470234-50701780-47ca-11e9-8021-fe4ccbee0ed5.gif)
This will initially show logs there.

Reply:
Why is this the right solution?

Reply:
So when page starts up or refreshes, logs will be highlighted.

Reply:
Initially, when it was set to `false`, when the page use to start-up it, automatically, data-viewer tab became the active one.

Reply:
@domoritz I have reverted the default state to `false`. Found another solution to fix the problem. :)